src/database/schema/UserSchema.ts

```diff
@@ -25,7 +25,7 @@ class UserSchema implements BaseDbSchema {
 
   bio: string;
 
-  isBanned: boolean;
+  isBanned: boolean | null;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `isBanned: boolean` to `isBanned: boolean | null` is a significant one. It introduces a third state (null) into what was previously a binary (true/false) property. This could potentially introduce bugs in parts of the code that expect `isBanned` to be strictly a boolean and do not handle the `null` case.
```diff
-  isBanned: boolean;
+  isBanned: boolean | null;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->